packages/react-kit/src/components/SteppableInput/SteppableInput.tsx

```diff
@@ -21,6 +21,8 @@ export type TPickedInputProps = Pick<
 
 export type TFullSteppableInputProps = TPickedInputProps & {
 	isDisabled?: TInputProps['isDisabled'];
+	isDisabledMinButton?: boolean;
+	isDisabledMaxButton?: boolean;
```

Comment:
It is not Max and Min, but Increment and Decrement in context of SteppableInput. And also I think from the perspective of the English language it is more correct `isIncrementButtonDisabled` than `isDisabledIncrementButton`